socialstar.rb

```diff
@@ -0,0 +1,263 @@
+    "Cars/Auto",
+    "Travel",
+    "Pets",
+    "Bridal/Event",
+    "Vlogging",
+    "Other"
+  ]
+
+  GENDERS = ['Male', 'Female']
+  PAYMENT_METHODS = ['ZipMark', 'PayPal']
+
+  field :country_code, :default => 'US'
+  field :date_of_birth, :type => DateTime
+  field :email, :default => ''
+  field :encrypted_password, :default => ''
+  field :fb_user_id
+  field :first_name
+  field :has_seen_tutorial, :type => Boolean, :default => false
+  field :gender
+  field :last_name
+  field :mixpanel_member_id
+  field :password
+  field :password_confirmation
+  field :provider
+  field :referral_url
+  field :social_media_access_tokens, :type => Hash, :default => {}
+  field :crowdtap_member_id, :type => BSON::ObjectId
+  field :terms_of_service, :type => Boolean, :default => true
+  field :ethnicity
+  field :parenting_status
+  field :relationship_status
+  field :address_state
+  field :zip_code
+  field :social_channels, :type => Hash, :default => {}
+  field :admin, :type => Boolean, :default => false
+  field :socialstar_verified, :type => Boolean, :default => false
+  field :payment_method, :default => 'ZipMark'
+  field :notes
+  field :blog_name,           :type => String
+  field :blog_url,            :type => String
+  field :blog_visitors,       :type => Integer
+  field :blog_payment,        :type => String
+  field :blog_categories,     :type => Array,  :default => []
+  field :facebook_username,   :type => String
+  field :facebook_followers,  :type => Integer
+  field :facebook_payment,    :type => String
+  field :instagram_username,  :type => String
+  field :instagram_followers, :type => Integer
+  field :instagram_payment,   :type => String
+  field :pinterest_username,  :type => String
+  field :pinterest_followers, :type => Integer
+  field :pinterest_payment,   :type => String
+  field :twitter_username,    :type => String
+  field :twitter_followers,   :type => Integer
+  field :twitter_payment,     :type => String
+  field :vine_username,       :type => String
+  field :vine_followers,      :type => Integer
+  field :vine_payment,        :type => String
+  field :youtube_channel,     :type => String
+  field :youtube_subscribers, :type => Integer
+  field :youtube_payment,     :type => String
+
+  enum :status, [:pending, :approved, :rejected, :accepted]
+
+  validates :email,    format: { with: /\A([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})\z/i,
+                                 message: "not a valid email" },
+                                 allow_nil: true
+  validates :zip_code, format: { with: /\A\d{5}$|^[A-Z|a-z]\d[A-Z|a-z] ?\d[A-Z|a-z]\d\z/i,
+                                 message: "not a valid zipcode" },
+                                 allow_nil: true
+  validates :parenting_status, inclusion: { in: Crowdtap.info_poll_responses["parenting_status"],
+                                            message: "Parenting status must be one of #{Crowdtap.info_poll_responses["parenting_status"].values.join(' ')}" },
```

Comment:
Line is too long. [158/80]<br>Prefer single-quoted strings inside interpolations.